agent-config/src/macosX64Test/kotlin/com/epam/drill/agent/configuration/provider/EnvironmentVariablesProviderTest.kt

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright 2020 - 2022 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.epam.drill.agent.configuration.provider
+
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class EnvironmentVariablesProviderTest {
+
+    @Test
+    fun `to parameter name upper case`() {

```

Comment:
why do we have separate converters for names from different sources
I mean, can't we implement these conversions/checks in a single place